Tests/Portal/src/pages/tigerpages/loginPage.ts

```diff
@@ -1,6 +1,6 @@
 import loginPageJson from '../../../resources/pageLocators/tigerLoginPage.json';
```

Comment:
File doesn't exist in master branch. Please update path.

Reply:
Hi @trinadhvialto @saikatbhowmick-vialto 
This Pr Against Tiger branch not main branch